src/pages/index.tsx

```diff
@@ -241,14 +242,14 @@ const BlockList: React.FC<BlockListProps> = ({ blocks }) => {
     },
   ];
   return (
-    <DetailsList
-      items={blocks.slice(0, -1)}
+    <ShimmeredDetailsList
+      setKey="set"
+      items={loading ? [] : blocks.slice(0, -1)}
```

Comment:
Is it okay we slice `blocks`? (of course, it had been written in the previous commit...)

Reply:
I couldn't understand the intention. so I leave it the same.

Reply:
👌